tests/runtime_tests/registry.cpp

```diff
@@ -618,7 +618,8 @@ TEST_CASE("list tests", "[registry]") {
                 CHECK(console.messages == contains_substring("how many templated lights"));
                 CHECK(console.messages == contains_substring("hidden test 1"));
             } else if (tag == "[wrong_tag]"sv || tag == "[.hidden]"sv) {
-                CHECK(console.messages.empty());
+                CHECK(console.messages.size() < 22u);
```

Comment:
`22u` is a bit unclear, perhaps worth defining `auto header = "Matching test cases:\n"sv;` and then simply `CHECK(console.message == header);` (`header.size()` = 21 by the way, not sure why 22?). 

But we could also leave it as it is now, since this will have to be reworked anyway once we add the count `"X matching test cases"`.

Reply:
to address this and the other comment I did create a variable at the top above the `if`/`else` chain, great idea! It can be used to check that the output _contains_ that header before entering any of the specific checks, and also make sure that the output contains _only_ that in the final case. Can add the footer to this when that is implemented.

I think the 22 must have come from measuring the string literal in my text editor.. counting the escaped character as two. Silly last minute change mistake.